pyflakes/checker.py

```diff
@@ -2062,13 +1997,9 @@ def addArgs(arglist):
             wildcard = getattr(node.args, arg_name)
```

Comment:
this shouldn't need getattr any more

Reply:
Please could you elaborate on how to change it?

Reply:
should be able to loop over `(node.varag, node.kwarg)` directly (or whatever the names are)

Reply:
https://github.com/hugovk/pyflakes/commit/02728fcf9b7dee33d9914ab09a05f8d7d4f9c54e fails like https://github.com/hugovk/pyflakes/actions/runs/404149955:

```
  File "/home/runner/work/pyflakes/pyflakes/pyflakes/checker.py", line 842, in __init__
    self.handleChildren(tree)
  File "/home/runner/work/pyflakes/pyflakes/pyflakes/checker.py", line 1288, in handleChildren
    self.handleNode(node, tree)
  File "/home/runner/work/pyflakes/pyflakes/pyflakes/checker.py", line 1335, in handleNode
    handler(node)
  File "/home/runner/work/pyflakes/pyflakes/pyflakes/checker.py", line 1957, in FUNCTIONDEF
    self.LAMBDA(node)
  File "/home/runner/work/pyflakes/pyflakes/pyflakes/checker.py", line 1984, in LAMBDA
    for wildcard in (node.vararg, node.kwarg):
AttributeError: 'FunctionDef' object has no attribute 'vararg'
```

Reply:
then I think this code is dead, those were the python2 attributes!

Reply:
Hmm, removing it like https://github.com/hugovk/pyflakes/commit/5a149a93b3581076ebd0a9490ab4cb141de2f641 fails like https://github.com/hugovk/pyflakes/actions/runs/404358215:

```
FAIL: test_annotationUndefined (test.test_doctests.TestUndefinedNames)
Undefined annotations.
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/runner/work/pyflakes/pyflakes/pyflakes/test/test_undefined_names.py", line 657, in test_annotationUndefined
    ''')
  File "/home/runner/work/pyflakes/pyflakes/pyflakes/test/test_doctests.py", line 51, in flakes
    return super().flakes(self.doctestify(input), *args, **kw)
  File "/home/runner/work/pyflakes/pyflakes/pyflakes/test/harness.py", line 36, in flakes
    {}'''.format(input, expectedOutputs, '\n'.join([str(o) for o in w.messages])))
AssertionError: Lists differ: [<class 'pyflakes.messages.UnusedImport'>,[37 chars]rt'>] != []

First list contains 2 additional elements.
First extra element 0:
<class 'pyflakes.messages.UnusedImport'>

+ []
- [<class 'pyflakes.messages.UnusedImport'>,
-  <class 'pyflakes.messages.UnusedImport'>] : for input:
def doctest_something():
    """
       
       >>> from abc import note1, note2, note3, note4, note5
       >>> def func(a: note1, *args: note2,
       ...          b: note3=12, **kw: note4) -> note5: pass
    """

expected outputs:
[]
but got:
(none):4:12 'abc.note2' imported but unused
(none):4:12 'abc.note4' imported but unused
```

Reply:
oh right, it's `(node.args.vararg, node.args.kwarg)` not `(node.vararg, node.kwarg)`

Reply:
Done, thanks!